test/runtests.jl

```diff
@@ -130,7 +131,7 @@ end
     # methods of seg
     @test width(seg) == 1.0
     @test middle(seg) == 1.5
-    @test linspace(seg, 10) == linspace(1.0, 2.0, 10)
+    @test linspace(seg, 10) == range(1.0, stop=2.0, length=10)
```

Comment:
since `linspace` is deprecated, I am not sure we should even define a method for it. perhaps a `range(::Segment; length)` method with obligatory length?

Reply:
Yep, good catch. So...
```julia
@test range(seg, 10) == range(1.0, stop=2.0, length=10)
```
?